lazy-pages/src/lib.rs

```diff
@@ -56,6 +56,10 @@ thread_local! {
 /// instruction, which cause signal. Now memory which this instruction accesses
 /// is not protected and with correct data.
 extern "C" fn handle_sigsegv(_x: i32, info: *mut siginfo_t, _z: *mut c_void) {
+    if !cfg!(unix) {
+        panic!("unsupported");
```

Comment:
Isn't it too hard to panic in such case? What about placing here `return` statement?

Reply:
This code shouldn't ever hit @grishasobol ?